packages/serverless/src/domain/use-case/transaction.ts

```diff
@@ -0,0 +1,102 @@
+import { isDatabaseOwner } from './database.js';
+import { redisQueue } from '../../helper/mq.js';
+import { ModelDbDeployTx, ModelDbSetting } from '@zkdb/storage';
+import ModelUser from '../../model/global/user.js';
+import { MinaNetwork } from '@zkdb/smart-contract';
+import { PublicKey } from 'o1js';
+
+const MINA_DECIMAL = 1e9;
+
+export type TransactionType = 'deploy' | 'rollup';
+
+export async function enqueueTransaction(
```

Comment:
What is the difference between enqueue and getTx does?
I see it both return the tx

Reply:
enqueue does not return the transaction, it sends it to message queue

Reply:
Currently, we have a db deploy that does 2 things for us. Why you need to separate